src/styles/variables.scss

```diff
@@ -0,0 +1,31 @@
+//colors
+$mint: #3AAFA9;
+$dark-green: #2B7A78;
+$light-green: #DEF2F1;
+$bg-color-light: #FEFFFF;
+$dark-grey: #17252A;
+
+//fonts
+$title: "Fira Sans", sans-serif;
+$main-font: "Source Sans Pro", sans-serif;
```

Comment:
why $title and $main-font have similar value but different naming?
could you stick to the same naming please?